digitalpy/core/telemetry/impl/opentel_metric_reader.py

```diff
@@ -1,6 +1,6 @@
 from opentelemetry.sdk.metrics.export import MetricReader as OpenTelMetricReaderImpl
 import json
-from digitalpy.telemetry.metrics_reader import MetricReader
+from digitalpy.core.telemetry.metrics_reader import MetricReader
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>I003 isort expected 1 blank line in imports, found 0